homeassistant/components/bluecurrent/const.py

```diff
@@ -0,0 +1,190 @@
+"""Constants for the BlueCurrent integration."""
+import logging
+
+from homeassistant.components.sensor import SensorEntityDescription
+
+DOMAIN = "bluecurrent"
+
+LOGGER = logging.getLogger(__package__)
+
+PLATFORMS = ["sensor"]
+CHARGE_POINTS = "CHARGE_POINTS"
+DATA = "data"
```

Comment:
By the looks of it, these are related to the WebSocket API response. Maybe they should be named to that effect, or if they're not really constants you need to even know about in the rest of the integration, they needn't be here?

(Perhaps they should live in bluecurrent-api?)